<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AveiClass</class>
 <widget class="QMainWindow" name="AveiClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>584</width>
    <height>352</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>584</width>
    <height>352</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>584</width>
    <height>352</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>音视频导入导出</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="startBtn">
    <property name="geometry">
     <rect>
      <x>384</x>
      <y>240</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>启动</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="roomLineEdit">
    <property name="geometry">
     <rect>
      <x>76</x>
      <y>23</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>r8</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="importGroupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>251</width>
      <height>121</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="title">
     <string>导入</string>
    </property>
    <widget class="QListWidget" name="importListWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>231</width>
       <height>81</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Courier New</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 170, 0);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <item>
      <property name="text">
       <string>简易接口导入</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>原始数据导入(音频PCM,视频YUV)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>编码数据导入(音频AAC,视频H264)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>RTSP导入</string>
      </property>
     </item>
    </widget>
   </widget>
   <widget class="QGroupBox" name="exportGroupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>251</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="title">
     <string>导出</string>
    </property>
    <widget class="QListWidget" name="exportListWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>231</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Courier New</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 170, 0);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="viewMode">
      <enum>QListView::ListMode</enum>
     </property>
     <property name="selectionRectVisible">
      <bool>false</bool>
     </property>
     <item>
      <property name="text">
       <string>导出到文件</string>
      </property>
      <property name="font">
       <font>
        <family>新宋体</family>
        <pointsize>10</pointsize>
       </font>
      </property>
     </item>
     <item>
      <property name="text">
       <string>导出到回调</string>
      </property>
      <property name="font">
       <font>
        <family>新宋体</family>
        <pointsize>10</pointsize>
       </font>
      </property>
     </item>
    </widget>
   </widget>
   <widget class="QLabel" name="roomLabel">
    <property name="geometry">
     <rect>
      <x>22</x>
      <y>22</y>
      <width>48</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>房间号：</string>
    </property>
   </widget>
   <widget class="QLabel" name="descLabel">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>20</y>
      <width>241</width>
      <height>181</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>仿宋</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>请输入已存在房间号，选择某个导入或导出示例，然后点击启动按钮。
如果房间不存在，请先点击安排房间。</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="schedulePushButton">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>22</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>安排房间</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>584</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>roomLineEdit</tabstop>
  <tabstop>schedulePushButton</tabstop>
  <tabstop>importListWidget</tabstop>
  <tabstop>exportListWidget</tabstop>
  <tabstop>startBtn</tabstop>
 </tabstops>
 <resources>
  <include location="avei.qrc"/>
 </resources>
 <connections/>
</ui>
